UPGRADE.md

```diff
@@ -1,5 +1,13 @@
 # Upgrade to 2.8
 
+## Deprecated usage of DB-generated UUIDs
+
+The format of DB-generated UUIDs is inconsistent across supported platforms and therefore is not portable. Some of the platforms produce UUIDv1, some produce UUIDv4, some produce the values which are not even UUID.
+
+Unless UUIDs are used in stored procedures which DBAL doesn't support, there's no real benefit of DB-generated UUIDs comparing to the application-generated ones.
+
+Use a PHP library (e.g. [ramsey/uuid](https://packagist.org/packages/ramsey/uuid)) to generate UUIDs on the application side.
```

Comment:
I would suggest the [UUID PECL](https://pecl.php.net/package/uuid) instead.
It less invasive and the code is much more simple.

Reply:
What do you mean it's less invasive?

Reply:
I got intrigued with the "less invasive" too :smile: but DBAL would simply handle a string so it doesn't really matter which lib/extension is being used.

I'd still recommend @ramsey's lib, though. It's better documented (IMO) and supports all UUID versions (even the namespaced ones)...

Reply:
I also don't understand the _less invasive_ part.
The referenced PECL extension requires setup by server admin, has no documentation on php.net, is unix-only and has last release in 2015.

Reply:
It has not been updated because it does not need to be. It just works. 

About less invasive, i may be unclear. Sorry. I don't like the PHP lib because there is so many code for a really simple thing.

Reply:
Applies also to core libs: 90%+ of the code is just to emulate userland behaviour. If you can avoid an extension for doing something that can be done in userland, then do avoid it.